desktop/libs/notebook/src/notebook/sql_utils.py

```diff
@@ -50,14 +51,33 @@ def get_statements(hql_query):
     })
   return statements
 
+def get_hplsql_statements(hplsql_query):
+  statements = []
+  statements.append(
+    {
+      'start': {
+        'row': 0,
+        'column': 0
+      },
+      'end': {
+        'row': 0,
+        'column': len(hplsql_query) - 1
+      },
+      'statement': strip_trailing_semicolon(hplsql_query.rstrip())
+    })
+  return statements
+
```

Comment:
Any chance to move the logic into `split_statements()`?
(logic can be a util get_hpql_statements() and be called there)

That way we don't duplicate anything and keep the names of the methods approriate.

Reply:
Didn't get any direct way to add in `split_statements()`, that was the first thought in my mind also.
because this function runs for both 'impala' and 'hive' so in sql_utils there is not any direct way to catch the dialect?